exercises/supervised-regression/ex_rnw/ex_research-group.Rnw

```diff
@@ -0,0 +1,49 @@
+A medical research group wants to train a supervised regression model to predict how long a patient stays in the hospital after admission (measured in days). They have gathered data on each patient‘s age (in years), systolic blood pressure (in mmHg, a continuous scale), and weight (in kg). At first, they believe all of these three features should be modeled to have a linear effect on the target variable. To construct their supervised regression model, they remember how most of such models can be described:
+\medbreak
+\textbf{Learning = Hypothesis Space + Risk + Optimization.}
+\medbreak
+With the first of the three components of learning in his mind, researcher 1 comes up with the following hypothesis space:
```

Comment:
Ich würd den researchern namen geben, z.B. Holger, Lisa und Son :) Also in dem Fall ", Researcher 1 (Holger)" und unten entsprechend